docs/articles/documentation/test-api/executing-code-in-the-browser/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+title: Executing Code in the Browser
+permalink: /documentation/test-api/executing-code-in-the-browser/
+---
+# Executing Code in the Browser
```

Comment:
Oooh... It will be painful. We have a bunch of common text.
Maybe we'll just rename it to `Selectors and Client Code`, `Observing Page State` or something?